doc/samples/why_new_template.rst

```diff
@@ -56,6 +59,60 @@ All below commands assumes you are in :file:`{NCS_PATH}/sidewalk/samples` and bu
 | Device under test (CLI)   |``west build -b nrf52840dk_nrf52840 sid_dut``            | ``west build -b nrf52840dk_nrf52840 template -- -DOVERLAY_CONFIG="overlay-dut.conf"``        |
 +---------------------------+---------------------------------------------------------+----------------------------------------------------------------------------------------------+
 
+User interface
+--------------
+
+The user interface was changed in Sidewalk template, to better support board with smaller amount of buttons and leds.
+
+**Buttons** changes:
+
++--------------------------------+--------------------------+-------------------------+
+| Description                    | Old                      | New                     |
```

Comment:
Description -> maybe it could be a "button action"?
+would it be correct to say "Old mapping"/"New mapping"?